docs/user-guide/advanced-configuration/mesh-ingress.md

```diff
@@ -0,0 +1,117 @@
+# Mesh Ingress
+
+The mesh ingress allows user to peer external execution and hop nodes into the AWX control plane.
+This guide focuses on how to enable and configure the mesh ingress.
+For more information about what is remote execution node and hop node and how to create them see AWX Documentation on external remote execution node and hop node. <TODO: add AWX doc link>
```

Comment:
@TheRealHaoLiu I think we still need to add a link here right?

Reply:
Link to AWX docs: https://ansible.readthedocs.io/projects/awx/en/latest/administration/instances.html#managing-capacity-with-instances